eraser-map.yaml

```diff
@@ -861,7 +861,6 @@ styles:
     icons:
         base: points
         texture: pois
-        interactive: true
```

Comment:
The parameters in the `draw` blocks are all parameters that can vary per-feature, while those for a `style` affect everything rendered with that style. Most of the time they are dealing with pretty different things, and currently I don't believe there's any overlap between them. Sometimes the distinction is reflective of underlying technical architecture and doesn't feel natural, which I'd like to address (an example is having to create a `style` for each unique `texture`, so that the texture is bound in GL when the style is rendered, but then the specific `sprite` that is used within that texture is determined per-feature in the `draw` group). The reason `interactive` is a `draw` parameter is because it's often desirable to set the interactivity based on feature criteria, rather than enabling it in a blanket fashion. That said, I can see how it might be useful to use some `style`-level parameters as default values that can be overridden in the `draw` groups. I'll make a note for future architecture/syntax discussion.